microsoft-edge/webview2/how-to/webdriver.md

```diff
@@ -64,9 +64,9 @@ At this point you should have WebView2 Runtime installed, built a WebView2 proje
        Download NuGet package  
     :::image-end:::  
     
-1.  Type `Selenium.WebDriver` in the search bar, choose **Selenium.WebDriver** from the results, and make sure to checkmark the box next to **include pre-release**.  On the right-hand side window, ensure the **Version** is set to **install 4.0.0-alpha04** or later and choose **Install**.  NuGet downloads Selenium to your machine.  
+1.  Type `Selenium.WebDriver` in the search bar, choose **Selenium.WebDriver** from the results, and make sure to checkmark the box next to **include pre-release**.  On the right-hand side window, ensure the **Version** is set to **install 4.0.0-alpha07** or later and choose **Install**.  NuGet downloads Selenium to your machine.  
```

Comment:
Can we update the Selenium references throughout this doc to point to 4.0.0 Beta 4 (the current latest)?
https://www.nuget.org/packages/Selenium.WebDriver/4.0.0-beta4

Reply:
replaced!